tests/eval_files/trio102_anyio.py

```diff
@@ -0,0 +1,28 @@
+import anyio
+
+
+async def foo4():
+    try:
+        ...
+    except ValueError:
+        await foo()  # safe
+    except anyio.Cancelled:
```

Comment:
I presume this will be updated in a future PR 👍

Reply:
Yeah, there is no `anyio.Cancelled`.

Reply:
The AnyIO way to do this is `except anyio.get_cancelled_exc_class():`.

Reply:
Yep, discussed in https://github.com/Zac-HD/flake8-trio/pull/120#discussion_r1095001688 and tracked in #122 🙂

Reply:
Yeah that check is to make sure it doesn't raise an error - regardless of if it exists as a library function or not. Can add a comment with #122